api/src/infrastructure/postgres-survey-data.repository.ts

```diff
@@ -135,4 +117,96 @@ SELECT * FROM survey.answers WHERE survey_id IN (SELECT survey_id FROM survey.an
 
     widget.data = widgetData;
   }
+
+  //   public async addSurveyDataToSections(
```

Comment:
Since we will be refactoring the whole computation process, this commented out code can stay for reference for the upcoming task